library/src/main/java/com/android/volley/Request.java

```diff
@@ -29,6 +29,8 @@
 
 import java.io.OutputStream;
 import java.io.UnsupportedEncodingException;
+import java.net.MalformedURLException;
+import java.net.URL;
```

Comment:
Not sure what the difference is, but other places in the app we use `android.net.Uri` and uri.getPath() instead of this URL class